components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -667,24 +762,40 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
             deallocate (RossbyRadiusTaper)
          end if
 
+        if (config_Redi_closure == 'N2_dependent') then
+           !$omp parallel
+           !$omp do schedule(runtime) private(k)
+           do iEdge = 1,nEdges
+              do k = minLevelEdgeBot(iEdge),maxLevelEdgeTop(iEdge)
+                 RediKappa(k,iEdge) = RediHorizontalTaper(iEdge)*gmKappaScaling(k,iEdge)*RediKappa(k,iEdge)
```

Comment:
I added the horizontal taper here as it will be the ramp, one, or rossby radius depending on parameters